packages/components/src/components/scroll-buttons/scroll-buttons.tsx

```diff
@@ -63,17 +88,26 @@ export class ScrollButtons {
         } else if (this.upButtonElement.disabled) {
             this.upButtonElement.disabled = false;
         }
-
         // Disable or enable the scroll down button
-        if (this.scrollContainerElementRef.scrollHeight - this.scrollContainerElementRef.scrollTop === this.scrollContainerElementRef.clientHeight) {
-            this.downButtonElement.disabled = true;
-        } else if (this.downButtonElement.disabled) {
+        // length 8 is just that maxiumum amount of element items visible without a scroll possibility
```

Comment:
But what if the element container is of a different size? This number 8 seems tied to the usage in the Map Template. This component should be able to work in other places too.